### PR TITLE
Preselect ClinVar ACMG criteria according to the variant's ACMG classification

### LINKED ISSUE #4874: ACMG classification on ClinVar form to reflect the classification of the variant
User request:

`On the page where we are supposed to enter the ACMG classification (1-5), I have noticed that the default choice is “Pathogenic” regardless of the actual classification. Since you also display information about the classification in Scout, perhaps this could also be reflected in the default suggestion in the dropdown list? I have made this very error, I accidentally reported a variant as P even though the Scout classification was LP.`

This was supposed to work as described in the request, but perhaps the functionality was changed or it's broken now?

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Closes #4874 

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html [hunk 1]
```diff
@@ -209,14 +209,20 @@ <h3 class="fs-subtitle">The mode of inheritance specific to the variant-disease
                 <legend class="text-dark">Germline Classification</legend>
                 <h3 class="fs-subtitle">Represents a variant-level classification for a disease, rather than an interpretation of the clinical significance of a variant for a specific patient.</h3>
 
+                {% set scout_classif = variant_data.var_obj.classification|replace("_"," ") if variant_data.var_obj.classification else "n.a." %}
```
Comment (dnil) [inline scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html:212]: 👍

File: scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html [hunk 2]
```diff
@@ -209,14 +209,20 @@ <h3 class="fs-subtitle">The mode of inheritance specific to the variant-disease
...
                     <span class="text-dark">Classification in Scout:</span><br>
-                    <span class="badge bg-secondary">{{variant_data.var_obj.classification|replace("_"," ")|upper if variant_data.var_obj.classification else "n.a."}}</span>
+                    <span class="badge bg-secondary">{{ scout_classif|upper }}</span>
                   </div>
                   <div class="col-8">
                     {{variant_data.var_form.clinsig.label(class="fw-bold, text-dark")}} <span class="text-danger" data-bs-toggle='tooltip' title="Required field"><strong>*</strong></span>
-                    {{variant_data.var_form.clinsig(class="form-control, btn-secondary")}}
+                    <select name="clinsig" class="form-control, btn-secondary">
+                      {% for classif, classif_value in variant_data.var_form.clinsig.choices %}
+                        <option value="{{classif_value}}" {% if classif==scout_classif|capitalize %} selected {% endif %}>{{classif}}</option>
```
Comment (dnil) [inline scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html:223]: Looks good to me! I haven't checked all classifications to see the capitalisation is correct (like "Variant of Unknown Significance") but I take it you did? 😊

**Review by dnil — APPROVED** [review]
Looks good, if it is enough with `capitalize` instead of doing the usual (upper or lower for both).

**Comment by northwestwitch** [comment]
### Pathogenic: OK

image

### Likely pathogenic: also OK

image

### Like benign: OK

image

### Benign also OK

image

Uncertain was tested in the PR description!

File: scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html (hunk 2 above)
Comment (northwestwitch) [reply scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html:223]: All 5, they're 5 no?

Comment (dnil) [reply scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html:223]: Right! 1, 2, many!

**Comment by northwestwitch** [comment]
> Looks good, if it is enough with `capitalize` instead of doing the usual (upper or lower for both).

Yeah it's the first letter!

**Pull request merged**